spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -856,6 +864,173 @@ def release!(app_id, version)
+
+    # Returns list of all available In-App-Purchases
+    def iaps(app_id: nil)
+      r = request(:get, "ra/apps/#{app_id}/iaps")
+      return r.body["data"]
+    end
+
+    # Returns list of all available Families
+    def iap_families(app_id: nil)
+      r = request(:get, "ra/apps/#{app_id}/iaps/families")
+      return r.body["data"]
+    end
+
+    # Deletes a In-App-Purchases
+    def delete_iap!(app_id: nil, purchase_id: nil)
+      r = request(:delete, "ra/apps/#{app_id}/iaps/#{purchase_id}")
+      handle_itc_response(r)
+    end
+
+    # loads the full In-App-Purchases
+    def load_iap(app_id: nil, purchase_id: nil)
+      r = request(:get, "ra/apps/#{app_id}/iaps/#{purchase_id}")
+      parse_response(r, 'data')
+    end
+
+    # loads the full In-App-Purchases-Family
+    def load_iap_family(app_id: nil, family_id: nil)
+      r = request(:get, "ra/apps/#{app_id}/iaps/family/#{family_id}")
+      parse_response(r, 'data')
+    end
+
+    # updates an In-App-Purchases-Family
+    def update_iap_family!(app_id: nil, family_id: nil, data: nil)
+      with_tunes_retry do
+        r = request(:put) do |req|
+          req.url "ra/apps/#{app_id}/iaps/family/#{family_id}/"
+          req.body = data.to_json
+          req.headers['Content-Type'] = 'application/json'
+        end
+        handle_itc_response(r.body)
+      end
+    end
+
+    # updates an In-App-Purchases
+    def update_iap!(app_id: nil, purchase_id: nil, data: nil)
+      with_tunes_retry do
+        r = request(:put) do |req|
+          req.url "ra/apps/#{app_id}/iaps/#{purchase_id}"
+          req.body = data.to_json
+          req.headers['Content-Type'] = 'application/json'
+        end
+        handle_itc_response(r.body)
+      end
+    end
+
+    def create_iap_family(app_id: nil, name: nil, product_id: nil, reference_name: nil, versions: [])
+      r = request(:get, "ra/apps/#{app_id}/iaps/family/template")
+      data = parse_response(r, 'data')
+
+      # data['activeAddOns'][0]['name'] = { value: name }
+      data['activeAddOns'][0]['productId'] = { value: product_id }
+      data['activeAddOns'][0]['referenceName'] = { value: reference_name }
+      data['name'] = { value: name }
+
+      versions_array = []
+      versions.each do |k, v|
+        versions_array << {
+                  value: {
+                    subscriptionName: { value: v[:subscription_name] },
+                    name: { value: v[:name] },
+                    localeCode: { value: k.to_s }
+                  }
+        }
+      end
+      data["details"]["value"] = versions_array
+
+      r = request(:post) do |req|
+        req.url "ra/apps/#{app_id}/iaps/family/"
+        req.body = data.to_json
+        req.headers['Content-Type'] = 'application/json'
+      end
+      handle_itc_response(r.body)
+    end
+
+    # returns pricing goal array
+    def iap_subscription_pricing_target(app_id: nil, purchase_id: nil, currency: "EUR", tier: 1)
```

Comment:
Should we have `EUR` the default? Let's make it a required parameter maybe. What is the currency used for anyway?